libs/components/src/next/form/datepicker/vl-datepicker.component.ts

```diff
@@ -0,0 +1,370 @@
+import { baseStyle, resetStyle } from '@domg/govflanders-style/common';
+import {
+    datepickerStyle,
+    iconStyle,
+    inputAddonStyle,
+    inputFieldStyle,
+    inputGroupStyle,
+    tooltipStyle,
+} from '@domg/govflanders-style/component';
+import datepickerUigStyle from './vl-datepicker.uig-css';
+import { CSSResult, html, TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { FormControl, FormControlDefaults } from '../form-control/FormControl';
+
+import { Options } from 'flatpickr/dist/types/options';
+import { Instance } from 'flatpickr/dist/types/instance';
+import flatpickr from 'flatpickr';
+
+import Dutch from 'flatpickr/dist/l10n/nl.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { registerWebComponents } from '@domg-wc/common-utilities';
+import { VlButtonInputAddon, VlIconElement } from '@domg-wc/elements';
+import { live } from 'lit/directives/live.js';
+
+export const DatepickerDefaults = {
+    ...FormControlDefaults,
+    block: false,
+    readonly: false,
+    type: '' as 'range' | 'time' | 'date-time',
+    value: '',
+    format: 'd.m.Y',
+    selectedDate: '',
+    minDate: '',
+    maxDate: '',
+    minTime: '',
+    maxTime: '',
+    amPm: '',
+    pattern: '',
+};
+
+@customElement('vl-datepicker-next')
+export class VlDatepickerComponent extends FormControl {
+    // Properties
+    private value = DatepickerDefaults.value;
+    private type = DatepickerDefaults.type;
+    private block = DatepickerDefaults.block;
+    private readonly = DatepickerDefaults.readonly;
+    private format = DatepickerDefaults.format;
+    private selectedDate = DatepickerDefaults.selectedDate;
+    private minDate = DatepickerDefaults.minDate;
+    private maxDate = DatepickerDefaults.maxDate;
+    private minTime = DatepickerDefaults.minTime;
+    private maxTime = DatepickerDefaults.maxTime;
+    private amPm = DatepickerDefaults.amPm;
+    private pattern = DatepickerDefaults.pattern;
+
+    // Variables
+    instance?: Instance;
```

Comment:
Hier zou ik `instance: Instance | null = null` van maken zoals we op andere plaatsen doen.

Reply:
Done.